java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -96,11 +103,26 @@
       ImmutableSet.of("control", "enabled", "disabled", "treatment", "treated");
 
   private static final int DONTCARE = -1;
```

Comment:
delete unused field?

Reply:
It's an existing field that's used. I didn't introduce it.